streams/src/test/java/org/apache/kafka/streams/integration/GlobalKTableIntegrationTest.java

```diff
@@ -261,16 +265,23 @@ public boolean conditionMet() {
         }, 30000L, "waiting for initial values");
         System.out.println("no failed test");
     }
+    
+    @Test(timeout = 10000)
+    public void shouldRestoreAbortedMessages() throws ExecutionException, InterruptedException {
+        produceAbortedMessages();
+        startStreams();        
+    }
 
     private void createTopics() throws InterruptedException {
         streamTopic = "stream-" + testNo;
         globalTableTopic = "globalTable-" + testNo;
         CLUSTER.createTopics(streamTopic);
         CLUSTER.createTopic(globalTableTopic, 2, 1);
     }
-
+    
```

Comment:
Please avoid unnecessary reformatting -- there is more in this PR -- guess, it's a matter of IDE settings?

Reply:
Yes, it's a matter of IDE settings, we don't need these chages.